build.sbt

```diff
@@ -1,7 +1,7 @@
 name := "media4s"
 organization := "org.matthicks"
 version := "1.0.14"
-scalaVersion := "2.13.0"
```

Comment:
Remove this. I don't want the default Scala version to change.

Reply:
Sure